xml/operations-configuring-identity-identity_limitations.xml

```diff
@@ -322,8 +322,17 @@ for production use until a future OpenStack release.</li> -->
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conforms to the OpenID Connect specification.
```

Comment:
Either say "with the Google OAuth 2.0 APIs, which conform" if it really is more than one API, or "with the Google OAuth 2.0 API, which conforms" if it is just one API.

Reply:
I think it's officially "Google OAuth 2.0 APIs", APIs which handle authentication and authorization. See
https://developers.google.com/identity/protocols/OpenIDConnect

Reply:
OK so then just change "conforms" to "conform".